app/views/compute_attributes/edit.html.erb

```diff
@@ -1,9 +1,24 @@
-<%= breadcrumbs do
-      [
-        {caption: _("Compute Resources"), url: (url_for(compute_resources_path) if authorized_for(hash_for_compute_resources_path))},
-        {caption: @set.compute_resource.to_label, url: (compute_resource_path(@set.compute_resource) if authorized_for(hash_for_compute_resource_path(@set.compute_resource)))},
-        {caption: _("Compute Profiles"), url: (url_for(compute_profiles_path) if authorized_for(hash_for_compute_profiles_path))},
-        {caption: _('Edit %s' % @set.compute_profile.to_label)}
-      ]
-    end %>
+<%= breadcrumbs(
+    items: [
+      {
+        caption: _("Compute Resources"),
+        url: (url_for(compute_resources_path) if authorized_for(hash_for_compute_resources_path))
+      },
+      {
+        caption: @set.compute_resource.to_label,
+        url: (compute_resource_path(@set.compute_resource) if authorized_for(hash_for_compute_resource_path(@set.compute_resource)))
+      },
+      {
+        caption: _("Compute Profiles"),
+        url: (url_for(compute_profiles_path) if authorized_for(hash_for_compute_profiles_path))
+      },
+      {
+        caption: _('Edit %s' % @set.compute_profile.to_label)
+      }
+    ],
+    resource_url:api_compute_resource_compute_profiles_path(@set.compute_resource),
+    switcher_item_url: "/compute_profiles/#{@set.compute_profile.id}/compute_attributes/:id/edit"
```

Comment:
@ohadlevy, @amirfefer and I have discussed about it.
Basically we agree, but because we are in a rush, and we really want to it to get merged, we will create a smaller PRs later in order to fix it.